packages/test/test-gc-sweep-tests/src/test/autoRunTest.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { rootDataObjectWithChildDataObjectFactory } from "../dataObjectWithChildDataObject";
+import { dataObjectWithCounterFactory } from "../dataObjectWithCounter";
+
+describeNoCompat("GC InactiveObjectX tests", (getTestObjectProvider) => {
+    let provider: ITestObjectProvider;
+
+    const summaryOptions = DefaultSummaryConfiguration as ISummaryConfigurationHeuristics;
+    summaryOptions.summarizerClientElection = true;
+    // Summaries should run automatically
+    const runtimeOptions: IContainerRuntimeOptions = {
+        summaryOptions,
+        gcOptions: {
+            gcAllowed: true,
+        },
+    };
+    const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+        runtime.IFluidHandleContext.resolveHandle(request);
+    const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
+        rootDataObjectWithChildDataObjectFactory,
+        [
+            [rootDataObjectWithChildDataObjectFactory.type, Promise.resolve(rootDataObjectWithChildDataObjectFactory)],
+            [dataObjectWithCounterFactory.type, Promise.resolve(dataObjectWithCounterFactory)],
+        ],
+        undefined,
+        [innerRequestHandler],
+        runtimeOptions,
+    );
+
+    // const sessionExpiryDurationMs = 3000; // 3 seconds
+    const inactiveTimeoutMs = 10 * 1000; // 10 seconds
+
+    // Set settings here, may be useful to put everything in the mockConfigProvider
+    const settings = {
+        // "Fluid.GarbageCollection.RunSessionExpiry": "true",
+        "Fluid.GarbageCollection.TestOverride.InactiveTimeoutMs": inactiveTimeoutMs,
+        // "Fluid.GarbageCollection.TestOverride.SessionExpiryMs": sessionExpiryDurationMs,
+    };
+    const configProvider = mockConfigProvider(settings);
+
+    let overrideLogger: IgnoreErrorLogger;
+
+    // Test timeout
+    const testTimeout = 5 * 60 * 1000; // 5 minutes
+
+    // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
+    // Note: can run with npm run test:build to build and run the test
+    // TODO: have this configurable via mocha cmd arguments
+    // TODO: setup test to run in CI
+    const numberOfTests = 1;
+    for (let i = 0; i < numberOfTests; i++) {
+        const seed = Math.random();
+        const liveContainers = 10;
+        const random = makeRandom();
+        it(`GC Ops Test with Seed: ${seed}, Containers: ${liveContainers}`, async () => {
+            provider = getTestObjectProvider({
+                syncSummarizer: true,
+            });
+
+            // Wrap the logger
+            overrideLogger = new IgnoreErrorLogger(provider.logger);
+            provider.logger = overrideLogger;
+
+            // Create the containerManager responsible for retrieving, creating, loading, and tracking the lifetime of containers.
+            const containerManager = new ContainerManager(runtimeFactory, configProvider, provider);
+            await containerManager.createContainer();
```

Comment:
Since we start with only 1, looks like for the first minute and a half we are building up from 1->10 containers.  Then for the remainder of the 5 min we alternate between 9-10.  Does that sound right?  Seems fine, but a little unexpected.

Reply:
Made the test start with 10 live containers